src/app/views/login/SignInController.test.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { LoginController } from "./SignInController";
+import renderer from "react-test-renderer";
+import { mount, shallow } from "enzyme";
+
+jest.mock("../../utilities/notifications.js", () => {
+    return {
+        add: function() {
+            // do nothing
+        }
+    };
+});
+
+jest.mock("../../utilities/websocket", () => {
+    return {
+        send: function() {
+            // do nothing
+        }
+    };
+});
+
+jest.mock("../../utilities/api-clients/user.js", () => {
+    return {
+        setUserState: function() {
+            // do nothing
+        },
+        logOut: function() {
+            // do nothing
+        }
+    };
+});
+jest.mock("../../utilities/redirectToMainScreen.js", () => {
+    return {
+        redirectToMainScreen: function() {
+            // do nothing
+        }
+    };
+});
+
+test("Check redirect doesn't work if not authenticated", () => {
+    const props = {
+        dispatch: function() {},
+        rootPath: "/florence",
+        isAuthenticated: false
+    };
+    const component = mount(<LoginController {...props} />);
+    expect(component.find("h1").text()).toBe("Log in to your Florence account");
```

Comment:
As per the description, this is mostly a duplicate file that will be worked on in future PRs.
- This line however is changed.